ZENOFCHAOS.md

```diff
@@ -0,0 +1,10 @@
+# Zen of Chaos.
+This is the Zen of Chaos. We pay tribute to the Chaos. The Chaos join us to accomplish its purpose.
+
+This is a list of principles that made The Chaos our religion.
+
+1. The Chaos is complicated
+2. The Chaos is our religion
```

Comment:
these should all be `1.`, that way it's easier to add/reorder them (better diffs), github will automatically number them.